myst_nbs/splines/spline.myst.md

```diff
@@ -6,17 +6,17 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: Python 3.9.10 ('pymc-dev-py39')
   language: python
   name: python3
 ---
 
 # Splines in PyMC3
 
-:::{post} Oct 8, 2021 
-:tags: patsy, pymc3.Deterministic, pymc3.Exponential, pymc3.Model, pymc3.Normal, regression, spline
+:::{post} May 6, 2022 
+:tags: patsy, pymc.Deterministic, pymc.Exponential, pymc.Model, pymc.Normal, regression, spline
```

Comment:
There should not be `pymc.<object>` tags anymore, that generates way too many tags that are not that useful so we decided to use https://docs.pymc.io/projects/examples/en/latest/object_index/index.html instead for this. The issue to track progress on this is https://github.com/pymc-devs/pymc-examples/issues/289